snippets/breadcrumbs.liquid

```diff
@@ -1,17 +1,61 @@
-<div class="breadcrumbs-container">
-  <div class="breadcrumbs-listings">
-    <div class="listings__item">
-      <a href="/" class="first-listing">{{ 'snippets.collection-listing.home' | t }}</a>
-    </div>
-    <div class="listings__item">
-      <img src='{{ 'left-arrow.png' | asset_url }}'>
-    </div>
-    <div class="listings__item">
-      <a href="/collections" class="second-listing">{{ 'snippets.collection-listing.collections' | t }}</a>
+{%- if template contains 'collection' or template contains 'list-collections' -%}
+  <div class="breadcrumbs-container">
+    <div class="breadcrumbs-listings">
+      <div class="listings__item">
+        <a href="/" class="first-listing">{{ 'snippets.collection-listing.home' | t }}</a>
+      </div>
+      <div class="listings__item arrow">
+        <img src='{{ 'left-arrow.png' | asset_url }}'>
+      </div>
+      <div class="listings__item">
+        <a href="/collections" class="second-listing">{{ 'snippets.collection-listing.collections' | t }}</a>
+      </div>
     </div>
+    {%- if template contains 'collection' -%}
+      <div class="breadcrumbs-name">
+      {%- dump collection.products -%}
+        <h1>{{ category.name }}</h1>
+      </div>
+      {%- else -%}
+      <div class="breadcrumbs-name">
+        <h1>{{ 'snippets.collection-listing.collections' | t }}</h1>
+      </div>
+    {%- endif-%}
   </div>
-
-  <div class="breadcrumbs-name">
-    <h1>{{ 'snippets.collection-listing.collections' | t }}</h1>
+{%- elsif template contains 'product' -%}
+  <div class="breadcrumbs-container">
+    <div class="breadcrumbs-listings">
+      <div class="listings__item">
+        <a href="/" class="first-listing">{{ 'snippets.collection-listing.home' | t }}</a>
+      </div>
+      <div class="listings__item arrow">
+        <img src='{{ 'left-arrow.png' | asset_url }}'>
+      </div>
+      <div class="first-listing">
+        {{ category.name }}
+      </div>
+      <div class="listings__item arrow">
+        <img src='{{ 'left-arrow.png' | asset_url }}'>
+      </div>
+      <div class="second-listing">
+        {{ product_name }}
+      </div>
+    </div>
   </div>
-</div>
+{%- else -%}
+  {% unless template contains 'index' %}
```

Comment:
you can remove this line because we don't render the breadcrumbs on the home page as well 
or you can add `{% unless template contains 'page' %}` for the other pages